GoogleChromeLabs/ps-analysis-tool
# Fix: Report generation

## Description
After adding global filtering in the CLI and the extension. It didnt make a point to download the dashboard landing page. So now if user downloads report from anywhere it will download the full report with the relevant data.

## Relevant Technical Choices
- Remove the usage of report package.

## Testing Instructions
For CLI:
- Clone this branch. In the terminal run `npm run build:all`.
- Now in CLI run `npm run cli -- -u https://bbc.com`
- Open the report and download the report from the CLI dashboard.
- It should open a similar report to the CLI-dashboard.

For extension:
- Open the browser and install the PSAT extension zip.
- Download the report from the browser.
- Then download the report again from the dashboard, it should be similar to the report downloaded.

## Additional Information:



## Screenshot/Screencast



---

## Checklist
